style.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap');
+
+body {
+  margin: 0;
+  padding: 24px;
+  font-family: Poppins, sans-serif;
+  display: flex;
+  flex-direction: column;
+  background-image: url(img/backg.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.bio-section {
```

Comment:
- [ ] Please I highly suggest adding CSS Flexbox for alignment of the `.bio-section` class.